docs/installation.md

```diff
@@ -49,7 +50,12 @@ Note that you will need to specify your UPPMAX project ID when running a pipelin
 params.project = 'project_ID' // eg. b2017123
 ```
 
-## 3.2) Configuration: Other clusters
+## 3.2) Configuration: Hebbe (Chalmers)
```

Comment:
Should probably change `Chalmers` to `C3SE` - comparable difference to `Uppmax` and `Uppsala university`